CHANGELOG.rst

```diff
@@ -62,6 +62,7 @@ CHANGELOG
 * Fix bug that was causing failures in sqswatcher when ADD and REMOVE event for the same host are fetched together.
 * Fix bug that was preventing nodes to mount partitioned EBS volumes.
 * Implement paginated calls in ``pcluster list``.
+* Fix bug when creating ``awsbatch`` cluster with name longer than 31 chars
```

Comment:
isn't it 46 chars?

Reply:
no, because we prepend "parallelcluster-" which is 15 chars
46 - 15 = 31